test-suite/js/src/test/scala/org/scalajs/testsuite/library/BigIntTest.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Assume._
+import org.junit.{ BeforeClass, Test }
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+
+object BigIntTest {
+  @BeforeClass
+  def assumeRuntimeSupportsBigInt(): Unit = {
+    assumeTrue(Platform.executingInNodeJS)
+    val bigIntConstructor =
+      js.Dynamic.global.BigInt.asInstanceOf[js.UndefOr[js.Any]]
+    assumeTrue(bigIntConstructor.isDefined)
+  }
+}
+
+class BigIntTest {
+
+  // Methods we actually implement
+
+  @Test def apply(): Unit = {
+    val bi = js.BigInt("9007199254740992")
+    assertEquals(bi.toString(), "9007199254740992")
```

Comment:
Please also test the other overloads of `apply()`.

Reply:
Also, you should swap the arguments to most of your calls to `assertEquals`. The meaning of the arguments is
```scala
assertEquals(expected, actual)
```
This is not just by convention; the error messages reported by JUnit only make sense if the first argument is the expected value, and the second argument is the actual value.